packages/admin-ui/src/calendar/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { useDateFormatter } from '../i18n'
+import { setDay } from 'date-fns'
+
+export interface WeekDay {
+  readonly title: string
+  readonly abbr: string
+}
+
+export function useWeekDays(weekStart: number): WeekDay[] {
+  const dayFormatter = useDateFormatter({ weekday: 'narrow' })
+  const dayFormatterLong = useDateFormatter({ weekday: 'long' })
+
+  return [0, 1, 2, 3, 4, 5, 6].map((index) => {
+    const dateDay = setDay(Date.now(), (index + weekStart) % 7)
+
+    const day = dayFormatter.format(dateDay)
+    const dayLong = dayFormatterLong.format(dateDay)
+
+    return { title: dayLong, abbr: day } as const
+  })
+}
+
+export function toUTCString(date: Date) {
+  return date.toISOString().slice(0, 10)
```

Comment:
**Suggestion:** 
Maybe it would be better to get `day`, `month`, `year` and only then return the date formatted. 
The function slice is not considering the timezone, so depending on timezone and the time you get that information, you could return `2022-02-17` or ` 2022-02-18`, but the right date would still be 2022-02-17. 🤔

Reply:
I'm solving on other PR (datepicker)